src/Files.App/UserControls/PathBreadcrumb.xaml

```diff
@@ -98,7 +98,7 @@
 								ScrollViewer.VerticalScrollBarVisibility="Auto"
 								ScrollViewer.VerticalScrollMode="Auto">
 								<MenuFlyout.MenuFlyoutPresenterStyle>
-									<Style TargetType="MenuFlyoutPresenter">
+									<Style BasedOn="{StaticResource DefaultMenuFlyoutPresenterStyle}" TargetType="MenuFlyoutPresenter">
```

Comment:
Isn't this automatic?

Reply:
Yeah, removed and added the property explicitly.

Reply:
I got the very important tip.
Lack of "Brush" make the brush red in HC theme.

Please note this for the further development on XAML.

Reply:
You're right.
Removed and added the property setter explicitly.